starlette_jsonapi/resource.py

```diff
@@ -46,6 +47,9 @@ class BaseResource(metaclass=RegisteredResourceMeta):
     # 'str', 'int', 'float', 'uuid', 'path'
     id_mask: str = 'str'
 
+    # Paginator class, subclass of BasePaginator
+    pagination_class: Optional[Type[BasePaginator]] = None
```

Comment:
Just curious, why not `paginator_class` ?

Reply:
I've made some changes to the naming. The pagination classes are referred to as `pagination `( i.e: BasePagination, BaseOffsetPagination, etc.) and instances of pagination classes are referred to as `paginator`